package.json

```diff
@@ -15,6 +15,7 @@
   "scripts": {
     "changelog:api": "cd api && yarn changelog",
     "changelog:app": "cd app && yarn changelog",
-    "changelog:extension": "cd extension && yarn changelog"
+    "changelog:extension": "cd extension && yarn changelog",
+    "lint:global": "cd api && yarn lint --fix && cd .. && cd app && yarn lint --fix && cd .. && cd extension && yarn lint:fix && cd .."
```

Comment:
do we really need this? will you want to lint all repos at once?

Reply:
yeah, why not. In global changes

Reply:
How come that this is merged and I don't have it when I do `git pull`?

Reply:
ah you revert it at some point right?